crates/bitwarden-exporters/src/cxf/import.rs

```diff
@@ -147,13 +149,16 @@ mod tests {
             id: [0, 1, 2, 3, 4, 5, 6].as_ref().into(),
             creation_at: Some(1706613834),
             modified_at: Some(1706623773),
-            ty: ItemType::Login,
             title: "Bitwarden".to_string(),
             subtitle: None,
             favorite: None,
             credentials: vec![],
             tags: None,
             extensions: None,
+            scope: Some(CredentialScope {
```

Comment:
Should this one also be None?

There's another two in this file, lines 204 and 278

Reply:
It should.